CarInspection/src/WebCarInspection/Models/DriverViewModel.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace WebCarInspection.Models
+{
+    public class DriverViewModel
+    {
+        public int Id { get; set; }
+
+        [Display(Name = "First name")]
```

Comment:
FYI:
In the best case, all these strings should be stored in constants fields.
But it's not important for now.